<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>News</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #222224</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>551</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
width: 75px;
height: 90px;
font-size: 14px;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(136, 138, 133);
font-size: 20px;
text-align: center;
color: white;</string>
   </property>
   <property name="text">
    <string> Input your news</string>
   </property>
  </widget>
  <widget class="QLabel" name="captionLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Serif&quot;;
border: 3px solid rgb(136, 138, 133);
color: white;</string>
   </property>
   <property name="text">
    <string>There will be status information</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
